src/main/java/de/tudarmstadt/ukp/jwktl/parser/de/components/nountable/PluralHandler.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright 2013
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package de.tudarmstadt.ukp.jwktl.parser.de.components.nountable;
+
+import java.util.regex.Matcher;
+
+import de.tudarmstadt.ukp.jwktl.api.entry.WiktionaryWordForm;
+import de.tudarmstadt.ukp.jwktl.api.util.GrammaticalNumber;
+import de.tudarmstadt.ukp.jwktl.parser.util.ParsingContext;
+
+public class PluralHandler extends PatternBasedIndexedParameterHandler {
+
+	protected static final String PLURAL_PATTERN =

```

Comment:
An alternative would be `\\s*Plural\s*([1-4])?\\?\\*{0,2}$` (with case insensitive flag), i.e., optional whitespace, the term "plural", optional whitespace, optional index number, optional ?, up to two  asterisks. This regex would be a bit more relaxed and thus keep working if the Wiktionary template evolves, e.g., allowing `Plural 2?`

Reply:
I a little bit dislike "relaxed" or "lenient" approach.
We're parsing a specific Wiktionary template here, according to the rules defined in the template. If the template evolves we should update the code. Defining a relaxed regex and hoping that it will handle future cases is, from my point of view, not the right approach. I think we should be strict here.

Being strict also helps to detect errors made by authors in Wiktionary articles. I've personally corrected a few dozen of a little bit incorrectly defined declination tables.

I'd like to leave `PLURAL_PATTERN` as is.